koku/api/query_handler.py

```diff
@@ -145,8 +145,6 @@ def get_resolution(self):
             (String): The value of how data will be sliced.
 
         """
-        if self.resolution:
-            return self.resolution
```

Comment:
Why is this being removed?

Reply:
Since I'm setting self.resolution in the __init__ method, the lines below (150-160) would not have been called resulting in attribute errors.